.github/workflows/build.yml

```diff
@@ -477,6 +481,7 @@ jobs:
     name: "Update manifest file on download server"
     runs-on: ubuntu-latest
     needs: build
+    if: always()
```

Comment:
this needs a comment, why is this necessary?

Reply:
Because otherwise it will not be executed if any of the previous `build` jobs fail. See https://docs.github.com/en/actions/using-jobs/using-jobs-in-a-workflow#example-not-requiring-successful-dependent-jobs.

Reply:
Comment added.

Reply:
Thank you, I figured it was something like that. Thank you for the comment. It should prevent people removing the `if: always()` as part of a drive-by cleanup.